en/ros2/px4_ros2_interface_lib.md

```diff
@@ -0,0 +1,365 @@
+# PX4 ROS 2 Interface Library
+
+The [PX4 ROS 2 Interface Library](https://github.com/Auterion/px4-ros2-interface-lib) is a C++ library to simplify controlling PX4 from ROS 2.
```

Comment:
FMI, is plan to move library into PX4 org?

Reply:
I don't know, it's not up to me.